.storybook/tsconfig.json

```diff
@@ -0,0 +1,35 @@
+{
+    "compilerOptions": {
```

Comment:
I'm thinking we can probably extend the `tsconfig` from the root of the project and then override any `compilerOptions` if needed.

Reply:
Yeah I tried to use ./tsconfig.json file but it wouldn't compile. Then I considered modifying it but didn't see any obvious way to do that since it took a filename as parameter, and it wasn't important enough to spend too much time investigating. I'm sure it's possible. The only compiler option that the tutorial called out is the `"jsx": "react",` which was different than the main one.